extension/src/experiments/columns/collect/index.ts

```diff
@@ -47,7 +47,9 @@ export const collectColumns = (data: ExperimentsOutput): Column[] => {
   for (const commit of Object.values(rest)) {
     collectFromCommit(acc, commit)
   }
-  return Object.values(acc)
+  const columns = Object.values(acc)
+
+  return columns.length === 1 ? [] : columns
```

Comment:
[Q] Would it be a good idea to explicitly check that `columns` is equal to only the timestamp column (`isEqual(columns, [timestampColumn])` ) and add an explanatory variable as to why we're removing it?

Reply:
Great idea!